### DISCUSSION TITLE
How to provide an ID > 0

### DISCUSSION THREAD

**Discussion by EarwainBanadar** [ZuluSCSI/ZuluSCSI-firmware#326] [Q&A]
Hello all,

the emulator shall provide a virtual drive with ID 1.
I'm providing a file HDA1.HDA, but this will be mapped to ID 0, which can be seen in the log file.

I'm getting an ID 1 only, if I'm also providing a HDA0.HDA file, but in this case there also is a ID 0, which in my case is conflicting with a physical SCSI drive of ID 0.

Is this a bug or a feature?

What can I do? 
Options in the ini file?

Best

**Comment by aperezbios** [comment]
I can't answer the question without seeing a log file. I'm not even sure I understand what it is you're reporting. Please supply a zululog file, @EarwainBanadar

**Comment by MeTalOneOEight** [comment]
He is basically saying the filename parsing for the ID gives a wrong ID.
If he names it HD1 the result is ID 0, and for HD0 he gets ID 1.
But since there's a hardware drive in the chain with ID 0 there's an ID conflict.

**Comment by aperezbios** [comment]
@EarwainBanadar it also appears that you aren't following the documented naming convention properly. You wrote:

"I'm getting an ID 1 only, if I'm also providing a HDA0.HDA file"

...but the _correct__ naming convention requires "HD" (no A) and then a number. This may be what's tripping you up, because the firmware expects the SCSI ID to be the third digit of the filename. In the case you described above, it is not.
